Makefile.am

```diff
@@ -266,7 +266,7 @@ functional-check:
 
 docs-coverage:
 	doxygen docs/Doxyfile || die; \
-	python -m coverxygen --xml-dir xml/ --src-dir ./ --output doc-coverage.info --exclude ".*/src/.*" --include ".*/src/lib/.*" --format json || die; \
+	python -m coverxygen --xml-dir xml/ --src-dir ./ --output doc-coverage.info --exclude ".*/src/globals.h" --exclude ".*/src/swupd_build_opts.h" --exclude ".*/src/swupd_build_variant.h" --exclude ".*/src/swupd_exit_codes.h" --exclude ".*/src/swupd.h" --exclude ".*/src/swupd_internal.h" --exclude ".*/src/manifest.h" --exclude ".*/src/swupd_curl_internal.h" --exclude ".*/src/swupd_curl.h" --format json || die; \
```

Comment:
I'm curious, how did you come up with the list of files to exclude?

Reply:
I excluded all files that aren't properly documented yet.

Reply:
ahh ok, so this patch is to fix some of the files but not all yet. Got it.